### ISSUE TITLE
--write mode can't resolve mount module

### ISSUE THREAD

**Issue by lucab85** [ansible/ansible-lint#3374]
##### Summary

The `--write` mode can't resolve `mount` module of the `ansible.posix` collection.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 6.15.0 using ansible 2.14.5
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

Execute the `--write` mode on the mount.yml playbook:

- mount.yml

```yaml
---
- name: Test playbook
  hosts: all
  tasks:
    - name: Mount
      mount:
```

##### Desired Behavior

I expect the `mount` playbook to look like this:

```yaml
---
- name: Test playbook
  hosts: all
  tasks:
    - name: Mount
      ansible.posix.mount:
```

##### Actual Behavior

Instead I obtain the following behavior:

```console (paste below)
 ansible-lint --write all mount.yml       
WARNING  Listing 1 violation(s) that are fatal
syntax-check[specific]: couldn't resolve module/action 'mount'. This often indicates a misspelling, missing collection, or incorrect module path.
mount.yml:5:7


                  Rule Violation Summary                   
 count tag                    profile rule associated tags 
     1 syntax-check[specific] min     core, unskippable    

Failed after : 1 failure(s), 0 warning(s) on 1 files.
```

**Comment by ajinkyau** [comment]
@lucab85 I am unable to reproduce this issue after the fix for #3344 was merged.
